static/js/global.d.ts

```diff
@@ -7,4 +7,15 @@ declare let zulip_test: any;
 
 interface JQuery {
     expectOne(): JQuery;
+    tab(opts: string): void; // From static/third/bootstrap
```

Comment:
Returns `this`, not `void`, and `opts` isn’t a good name for a `string`.

```suggestion
    tab(action?: "show"): this; // From static/third/bootstrap
```

(Based on `@types/bootstrap`, which we could consider pulling in instead.)

Reply:
Changed. It would be great if we can just use `@types/bootstrap` here, but don't you think that the types might have changed for most functions? I couldn't find v2.3.2 in the npm package.